docker/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+data:
+    image: busybox
+    volumes:
+        - ../:/var/www/app
+
+nginx:
+    image: yappabe/nginx
+    links:
+        - php
+    volumes_from:
+        - data
+    environment:
+        DOCUMENT_ROOT: /var/www/app/web
+        INDEX_FILE: app_dev.php
+        PHP_FPM_SOCKET: php:9000
+        DNSDOCK_ALIAS: sylius.docker
+
+mysql:
+    image: yappabe/mysql
+    environment:
+        MYSQL_PASS: sylius
+        MYSQL_USER: sylius
+        ON_CREATE_DB: sylius
+        DNSDOCK_ALIAS: mysql.sylius.docker
+
+php:
+    image: yappabe/php
+    working_dir: /var/www/app
+    volumes_from:
+        - data
+    links:
+        - mysql
+    mem_limit: 2000000000
+
+mailcatcher:
+    image: yappabe/mailcatcher
+    environment:
+        DNSDOCK_ALIAS: mailcatcher.sylius.docker
+
+# password: secret
+# vnc port 5999
+selenium:
```

Comment:
Using Firefox for our `javascript` Behat suite is not the best idea - I tried a lot of options while I was integrating JS tests with Travis and Chrome is the best available choice.